src/core_plugins/kibana/public/visualize/editor/editor.js

```diff
@@ -287,9 +287,9 @@ function VisEditor($scope, $route, timefilter, AppState, $window, kbnUrl, courie
     delete savedVis.savedSearchId;
     parent.set('filter', _.union(searchSource.getOwn('filter'), parent.getOwn('filter')));
 
-    // copy over all state except "aggs" and filter, which is already copied
+    // copy over all state except "aggs", "query" and "filter"
```

Comment:
I thought we could ditch it entirely but it turns out we still need some things, like the index.